rpc/common/service.pb.go

```diff
@@ -922,6 +922,7 @@ type PolicyMetadata struct {
 	unknownFields protoimpl.UnknownFields
 
 	Id                 string   `protobuf:"bytes,1,opt,name=id,proto3" json:"id,omitempty"`
+	AdvId              string   `protobuf:"bytes,2,opt,name=adv_id,json=advId,proto3" json:"adv_id,omitempty"`
```

Comment:
I recall removing this since 

1. It had a typo so I assumed no one probably was actively using it. 
2. I don't think we'll use it given the move to only use ID fields. 
 
Any reason why you brought it back?